### DISCUSSION TITLE
多lang同时检测

### DISCUSSION THREAD

**Discussion by dhl961123** [PaddlePaddle/PaddleOCR#12700] [Q&A] [answered]
paddleocr --image_dir C:\Users\huili.dong\Desktop\第一轮测试复核.pdf --use_gpu false --page_num 5 --lang='ch,en'
报错:
 File "D:\Python\Miniconda\envs\paddle_env\lib\site-packages\paddleocr\paddleocr.py", line 468, in parse_lang
    assert lang in MODEL_URLS['OCR'][DEFAULT_OCR_MODEL_VERSION][
AssertionError: param lang must in dict_keys(['ch', 'en', 'korean', 'japan', 'chinese_cht', 'ta', 'te', 'ka', 'latin', 'arabic', 'cyrillic', 'devanagari']), but got 'ch,en'
请问如何修改

**Comment by ericyeyeye** [comment, marked as answer]
> paddleocr --image_dir C:\Users\huili.dong\Desktop\第一轮测试复核.pdf --use_gpu false --page_num 5 --lang='ch,en' 报错: File "D:\Python\Miniconda\envs\paddle_env\lib\site-packages\paddleocr\paddleocr.py", line 468, in parse_lang assert lang in MODEL_URLS['OCR'][DEFAULT_OCR_MODEL_VERSION][ AssertionError: param lang must in dict_keys(['ch', 'en', 'korean', 'japan', 'chinese_cht', 'ta', 'te', 'ka', 'latin', 'arabic', 'cyrillic', 'devanagari']), but got 'ch,en' 请问如何修改

最後--lang='ch,en' 改成 --lang='ch'，ch中包含英文了，一次只能選定一種語言
`paddleocr --image_dir C:\Users\huili.dong\Desktop\第一轮测试复核.pdf --use_gpu false --page_num 5 --lang='ch'`
